README.md

```diff
@@ -255,7 +255,7 @@ docker run \
 #### Submitting inference requests
 General examples for submitting inference requests to a Triton server are
 available
-[here](https://github.com/triton-inference-server/server/blob/master/docs/client_examples.md).
+[here](https://github.com/triton-inference-server/client/#triton-client-libraries-and-examples).
 For convenience, we provide the following example code for using the Python
```

Comment:
I think that a mention of  the example file `qa/L0_e2e/test_model.py` can also be added, like in line 106 of this Readme

Reply:
My only hesitation on this is that we're not really maintaining that as a user-focused file. The model generation script has some use to end-users, since it's always handy to be able to whip up an example for testing, but I'd rather not have users adopt any of the weird things we do exclusively for testing in that file. Maybe instead we can work on getting together the next example notebook that we have planned and add a link to that once it's available?

Reply:
True if it's not made to be an example then it's better that way.